ci/build-travis-win.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+export PATH="$PATH:/c/Program Files (x86)/Microsoft Visual Studio/Installer/"
+export MSVC=$(vswhere -latest -products "*" -requires Microsoft.Component.MSBuild -property installationPath)
+export MSVC=$(echo /$MSVC | sed -e 's/\\/\//g' -e 's/://')
+export PATH="$PATH:$MSVC/MSBuild/15.0/Bin/"
+
+NANO_ROOT=`PWD`
+mkdir build_win
+cd build_win
+
+cmake \
+    -G'Visual Studio 15 2017 Win64' \
+    -DACTIVE_NETWORK=nano_test_network \
+    -DNANO_TEST=ON \
+    -DCMAKE_BUILD_TYPE=Debug \
+    -DCMAKE_VERBOSE_MAKEFILE=ON \
+    -DBOOST_ROOT="~/platform/boost/" \
+    -DIPHLPAPI_LIBRARY="/c/Program Files (x86)/Windows Kits/10/Lib/10.0.17134.0/um/x64/iphlpapi.Lib" \
```

Comment:
yeah, that's the fun of using ci it would seem, different patch levels. Travis is referencing it's latest available in the image, and appveyor was initially configured with what was latest at the time for it